app/frontend/src/assets/scss/style.scss

```diff
@@ -437,6 +437,13 @@ a,
     color: $bcgov-error;
   }
 
+  // hide asterisks in after input in Data Grid becasue they appear after column headers
+  .formio-component-datagrid {
+    label.field-required:after {
```

Comment:
Should psuedo elements like hover and after and stuff use `::`?
https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements

or does it work either way in SCSS